runtimes/dotnet-6.0/src/RuntimeRequest.cs

```diff
@@ -2,13 +2,32 @@ namespace DotNetRuntime
 {
 	public class RuntimeRequest
 	{
-		public string Payload { get; set; } = "";
+		public string BodyString { get; set; }
+		public object Body { get; set; }
+		public Dictionary<string, string> Headers { get; set; }
+		public string Method { get; set; }
+		public string Url { get; set; }
+		public string Host { get; set; }
+		public string Scheme { get; set; }
+		public string Path { get; set; }
+		public string QueryString { get; set; }
+		public Dictionary<string, string> Query { get; set; }
+		public int Port { get; set; }
 
-		public Dictionary<string, string> Variables { get; set; } =
-			new Dictionary<string, string>();
-
-		public Dictionary<string, string> Headers { get; set; } =
-			new Dictionary<string, string>();
+		public RuntimeRequest(string BodyString, object Body, Dictionary<string, string> Headers, string Method, string Url, string Host, string Scheme, string Path, string QueryString, Dictionary<string, string> Query, int Port)
```

Comment:
Let's be consistent across all runtimes with param order, following how they appear in an actual request: method, scheme, host, port, path, query, headers, body